source/puppet/4.7/reference/lang_defined_types.markdown

```diff
@@ -36,18 +36,26 @@ Defines can be used as simple macros or as a lightweight way to develop fairly s
 
 You can use a `define` statement to create a new defined resource type.
 
-``` puppet
+```puppet
 # /etc/puppetlabs/puppet/modules/apache/manifests/vhost.pp
-define apache::vhost (Integer $port, String[1] $docroot, String[1] $servername = $title, String $vhost_name = '*') {
-  include apache # contains Package['httpd'] and Service['httpd']
-  include apache::params # contains common config settings
+define apache::vhost (
+  Integer $port,
+  String[1] $docroot,
+  String[1] $servername = $title,
+  String $vhost_name = '*',
+) {
+  include ::apache # contains package['httpd'] and service['httpd']
+  include ::apache::params # contains common config settings
```

Comment:
The leading :: isn't needed for this version of Puppet... but I don't want to keep making you go back and forth. I'll trim that after I merge.